packages/types/src/extensions/checksum.ts

```diff
@@ -20,13 +20,22 @@ export interface ChecksumAlgorithm {
   checksumConstructor(): ChecksumConstructor | HashConstructor;
 }
 
+/**
+ * @internal
+ */
 export interface ChecksumConfiguration {
   addChecksumAlgorithm(algo: ChecksumAlgorithm): void;
   checksumAlgorithms(): ChecksumAlgorithm[];
 
+  /**
+   * @deprecated unused.
+   */
   [other: string | number]: any;
```

Comment:
I removed this in the move. What was the reason for these unbounded set of keys?